__tests__/app.test.js

```diff
@@ -46,3 +46,42 @@ describe("/api/categories", () => {
     });
   });
 });
+
+describe("/api/reviews/:review_id", () => {
+  describe("GET", () => {
+    test("responds with review object with correct properties and status 200", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review.review_id).toBe(1);
+          expect(body.review.title).toBe("Agricola");
```

Comment:
to allow for later extension of the endpoint, the test should use a flexible matcher eg expect.objectContaining or .toMatchObject in this case :) You can still however use .toBe for the review ID as this will remain unchanged